dashboard/app/views/shared/_course_wide_block.haml

```diff
@@ -18,7 +18,7 @@
 
 -elsif id == '20hour'
   .span6.miniblock
-    =link_to script_url(1) do
+    =link_to script_url('20-hour') do
```

Comment:
Can't use `Script::TWENTY_HOUR_NAME` here because it throws a circular dependency exception.  My guess is that's why it was `1` before instead of `Script::TWENTY_HOUR_ID`.